src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,12 @@
+appenders=kruize
+
+appender.kruize.type = Console
+appender.kruize.name = myOutput
```

Comment:
Can this be more descriptive ? What exactly does `myOutput` signify ?

Reply:
Its the name of the appender for the logger to get a reference. Changed its name to kruizeAppender.